lib/helper/Playwright.js

```diff
@@ -542,7 +543,7 @@ class Playwright extends Helper {
     this.browser.on('targetchanged', (target) => {
       this.debugSection('Url', target.url());
     });
-    this.browserContext = await this.browser.newContext({ acceptDownloads: true, ...this.options.emulate });
+    this.browserContext = await this.browser.newContext({ ignoreHTTPSErrors :this.options.ignoreHTTPSErrors,acceptDownloads: true, ...this.options.emulate });//Adding the HTTPSError ignore in the context so that we can ignore those errors
```

Comment:
```suggestion
    this.browserContext = await this.browser.newContext({ ignoreHTTPSErrors: this.options.ignoreHTTPSErrors,acceptDownloads: true, ...this.options.emulate });//Adding the HTTPSError ignore in the context so that we can ignore those errors
```

Reply:
please fix formatting issues like this one - `npm run lint-fix`

Reply:
Is it for me ??

Reply:
@DavertMik I have committed the change requested